### PR TITLE
add titles to toolbar links

### PR DESCRIPTION
**Pull request by ethn**

### CODE REVIEW

File: card/mod/05_standard/set/all/rich_html/toolbar.rb [hunk 1]
```diff
@@ -162,20 +162,20 @@ def close_link css_class
...
 
-  def toolbar_button text, symbol, hide, target
-    btn_class = 'btn btn-default'
+  def toolbar_button text, symbol, hide, tag_args
+    tag_args[:class] = [ tag_args[:class], 'btn btn-default' ].compact * ' '
+    tag_args[:title] ||= text
     link_text = "#{glyphicon symbol}<span class='menu-item-label #{hide}'>#{text}</span>"
 
-    if target[:page]
-      card_link target[:page], :class=>btn_class, :text=>link_text, :path_opts=>{:slot=>{:show=>:toolbar}}
```
Comment (ethn) [inline card/mod/05_standard/set/all/rich_html/toolbar.rb]: unless I'm mistaken, we don't ever really want the "show toolbar" here.

File: card/mod/05_standard/set/all/rich_html/menu.rb [hunk 2]
```diff
@@ -2,7 +2,7 @@
   view :menu, :denial=>:blank, :tags=>:unknown_ok do |args|
     return _render_template_closer if args[:menu_hack] == :template_closer
     return '' if card.unknown?
-    wrap_with :div, :class=>'menu-slot' do
+    wrap_with :div, :class=>'menu-slot nodblclick' do
```
Comment (ethn) [inline card/mod/05_standard/set/all/rich_html/menu.rb]: was finding double clicks on menu led to edit view

Comment (GerryG) [inline card/mod/05_standard/set/all/rich_html/menu.rb]: Isn't that a good thing?  That would be selecting the first menu item, no?  Seems like what you would want that action to do.

Comment (ethn) [inline card/mod/05_standard/set/all/rich_html/menu.rb]: makes rational sense, but it was confusing, even for me.   If I want to double-click, I can usually do it pretty much anywhere else.

**Pull request merged**